bin/utility.py

```diff
@@ -212,14 +220,18 @@ def validate_api_key(api_key, logger=None):
         if response_code == 401:
             return (False, 'Unauthorized. Please check your API key.')
         else:
-            # Need to handle this, as splunklib is unable to handle the exception with (400, {'error': 'error_reason'}) format
-            msg = "The API call to the GreyNoise platform have been failed with status_code: {} and error: {}".format(response_code, response_message['error'] if isinstance(response_message, dict) else response_message)
+            # Need to handle this, as splunklib is unable to handle the exception with
+            # (400, {'error': 'error_reason'}) format
+            msg = ("The API call to the GreyNoise platform have been failed "
```

Comment:
`The API call to the GreyNoise API has failed...`

Reply:
Resolved.